pkg/corerp/frontend/controller/applications/graph_util.go

```diff
@@ -248,8 +252,12 @@ func computeGraph(applicationName string, applicationResources []generated.Gener
 			applicationGraphResource.ProvisioningState = &state
 		}
 
-		connections := connectionsFromAPIData(resource, resources)          // Outbound connections based on 'connections'
-		connections = append(connections, providesFromAPIData(resource)...) // Inbound connections based on 'provides'
+		// Resolve Outbound connections based on 'connections'.
+		connections := resolveConnections(resource, connectionsPath, connectionsResolver(resources))
```

Comment:
Should we be checking the resource type here? It looks like all of these "resolvers" apply to all resource types. So if a resource has a property `.properties.connections` it works like `Applications.Core/containers`, has the same schema, etc.

Reply:
This is similar to the design I was talking about here: https://github.com/radius-project/radius/pull/7079/files#r1467935133 but without being driven by Capabilities.

Reply:
I'm good with this for now 👍 we will revisit when we do extensibility.